fpga_interchange/device_resources.py

```diff
@@ -729,7 +729,7 @@ def get_site_pin(self, site, site_pin_index):
 
         """
         tile = self.device_resource_capnp.tileList[site.tile_index]
-        tile_type_index = self.tile_type_to_idx[tile.type]
+        tile_type_index = tile.type
```

Comment:
And also this.

Reply:
https://github.com/SymbiFlow/python-fpga-interchange/pull/5#discussion_r541451871